.github/workflows/push-main.yml

```diff
@@ -78,21 +98,30 @@ jobs:
           service_account: ${{env.IAM_SERVICE_ACCOUNT}}
 
       - name: ✍🏽 Login to GAR using skopeo
+        env:
+          SKOPEO_AUTH_DIR: ${{env.SKOPEO_AUTH_DIR}}
+          GAR_IMAGE_REGISTRY: ${{env.GAR_IMAGE_REGISTRY}}
         run: |
-          sudo skopeo login -u oauth2accesstoken --password=${{ steps.auth.outputs.access_token }} ${{env.GAR_IMAGE_REGISTRY}}
+          sudo -E skopeo login -u oauth2accesstoken --password=${{ steps.auth.outputs.access_token }} $GAR_IMAGE_REGISTRY
```

Comment:
`sudo -E` is necessary to leverage [SKOPEO_AUTH_DIR](https://github.com/gitpod-io/workspace-images/pull/1128/files?diff=split&w=1#diff-2784b9c374ced1e3330a88b99fc29d7a5981296ecf89649161a9aaaf4ea5b25eR102), which is used [here](https://github.com/gitpod-io/workspace-images/pull/1128/files?diff=split&w=1#diff-2784b9c374ced1e3330a88b99fc29d7a5981296ecf89649161a9aaaf4ea5b25eR67)

Reply:
This same change is working well for the `dockerhub-release.yml` action